constants/__init__.py

```diff
@@ -27,14 +30,63 @@
 SUBNET_UID = 9
 # The root directory of this project.
 ROOT_DIR = Path(__file__).parent.parent
-# The maximum bytes for the hugging face repo (5 Gigabyte).
-MAX_HUGGING_FACE_BYTES = 5 * 1024 * 1024 * 1024
-# A mapping of block numbers to the max model size as of that block.
-# This dictionary must remain ordered by key.
-MAX_MODEL_PARAMETER_SIZES = [
-    (0, 186_000_000),
-    (2_405_920, 772_000_000),
+# Block at which 7b models, 2048 sequence lengths, bfloat16, and flash attention are used.
+BLOCK_7B = 9_999_999
+SEQUENCE_LENGTH_1 = 1024
+SEQUENCE_LENGTH_2 = 2048
+# A mapping of block numbers to the supported model types as of that block.
+ALLOWED_MODEL_TYPES_1 = {
+    GPT2LMHeadModel,
+    MistralForCausalLM,
+    LlamaForCausalLM,
+    BartForCausalLM,
+    FalconForCausalLM,
+    GPTNeoXForCausalLM,
+    GPTJForCausalLM,
+}
+ALLOWED_MODEL_TYPES_2 = {
+    MistralForCausalLM,
+    LlamaForCausalLM,
+    BartForCausalLM,
+    FalconForCausalLM,
+    GPTNeoXForCausalLM,
+    PhiForCausalLM,
+    GemmaForCausalLM,
+}
+# A mapping of block numbers to ModelParameters.
+MODEL_PARAMETERS_BY_BLOCK = [
+    (
```

Comment:
nit: Since we're past this, can we just omit this entry?

Reply:
Shows history a little bit but agreed we can omit this one.

Reply:
Removed.